src/apps/moola/plugin.e2e.ts

```diff
@@ -0,0 +1,12 @@
+import plugin from './plugin'
+
+describe('getPositionDefinitions', () => {
+  it('should get the address definitions successfully', async () => {
+    const positions = await plugin.getPositionDefinitions(
+      'celo',
+      '0x2f43b69f4ea32be168f19b998aaf5c3d87dbbdd6',
```

Comment:
I've updated our test account (used in other plugins) to have some debt too.
I'll share it with you too. Makes it easy for us to test all plugins.
```suggestion
      '0x2b8441ef13333ffa955c9ea5ab5b3692da95260d',
```

Reply:
Thanks!